stdlib/list.gr

```diff
@@ -149,21 +171,26 @@ let filteri = (fn, list) => {
 let rec reject = (fn, list) => {
   match (list) {
     [] => [],
-    [first, ...rest] => if (!fn(first)) [first, ...reject(fn, rest)] else reject(fn, rest)
+    [first, ...rest] =>
+      if (!(fn(first))) {
```

Comment:
Interesting that we got parens around the function call here.

Reply:
Hmmm, again I did that for a reason in some case but I can't remember, so I can remove it and run the formatter again

Reply:
Removed for now, let's see if we get any problems

Reply:
Remembered it was needed around infix expressions, but fixed.